src/main/java/me/ryanhamshire/GriefPrevention/PlayerEventHandler.java

```diff
@@ -1108,19 +1108,20 @@ public void onPlayerTeleport(PlayerTeleportEvent event)
 
         //FEATURE: prevent players from using ender pearls to gain access to secured claims
         TeleportCause cause = event.getCause();
-        if ((cause == TeleportCause.CHORUS_FRUIT && instance.config_claims_enderPearlsRequireAccessTrust) || (cause == TeleportCause.ENDER_PEARL && instance.config_claims_enderPearlsRequireAccessTrust))

```

Comment:
Was thinking more of `if (cause == instance.config_claims_enderPearlsRequireAccessTrust && (TeleportCause.CHORUS_FRUIT || TeleportCause.ENDER_PEARL))`

Reply:
idk if this or your first commit is more preferable though. I'm assuming performance is negligible, but if I were reading this logically to troubleshoot or such, I think this is more readable?

Reply:
Thats my first commit, I'd recommend checking my next one, which is more readable. But I'm assuming you meant

```java
if (instance.config_claims_enderPearlsRequireAccessTrust && (TeleportCause.CHORUS_FRUIT || TeleportCause.ENDER_PEARL))
```

Which my 2nd commit and your implementation has the exact same performance, no performance difference.
I do think my 2nd commit is more readable

Reply:
I'm not a fan of nested if's.